src/definitions/behaviors/api.js

```diff
@@ -157,11 +157,6 @@
                 },
                 write: {
                     cachedResponse: function (url, response) {
-                        if (response && response === '') {
```

Comment:
as discussed in https://github.com/fomantic/Fomantic-UI/pull/2588#discussion_r1040967294